### DISCUSSION TITLE
Allow sharing dashboards between organizations

### DISCUSSION THREAD

**Discussion by Carolina0802** [grafana/grafana#57000] [Dashboards, Panels & Visualizations]
Allow sharing dashboards between Grafana organizations to edit the dashboards once and see it reflected in all organizations (clients).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
